packages/types/src/index.ts

```diff
@@ -332,6 +353,13 @@ export interface MongoshBusEventsMap {
   'mongosh-sp:reset-connection-options': () => void;
   /** Signals that an optional dependency of the mongodb package is missing. */
   'mongosh-sp:missing-optional-dependency': (ev: SpMissingOptionalDependencyEvent) => void;
+
+  /** Signals that a run edit command has started executing. */
+  'mongosh-editor:run-edit-command': (ev: EditorRunEditCommandEvent) => void;
+  /** Signals that a run edit command has failed. */
+  'mongosh-editor:read-vscode-extensions-done': (ev: EditorReadVscodeExtensionsDoneEvent) => void;
+  /** Signals that reading vscode extensions from disc failed. */
+  'mongosh-editor:read-vscode-extensions-failed': (ev: EditorReadVscodeExtensionsFailedEvent) => void;
```

Comment:
I think we want to add these to `setup-logger-and-telemetry` in the cli-repl package so that we get log events from this :)